### DISCUSSION TITLE
Dashboard User Permission Control

### DISCUSSION THREAD

**Discussion by acmookey** [emqx/emqx#7933] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6263624/168046778-3dcd125d-fb64-4317-92ad-ec92f89ec83f.png)

I noticed in EMQX(v4.3), I can add dashboard user through web UI like the image above. I got two questions(any help is good):
1、What's the difference between administrator and viewer for remark? (the remark field is confused)
2、Is there any more detailed documentation about permission control for the dashboard?

**Comment by lafirest** [comment, marked as answer]
@acmookey  Hi, there is no permission control for the dashboard, it is in planning

**Comment by zmstone** [comment]
The remarks is only descriptive information so far.
Role-based access control is a feature planed for future 5.x releases, but it’s hard to say the delivery timeline now
